EnglishFilter/sections/antiadblock.txt

```diff
@@ -545,6 +545,9 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74503
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=hotcleaner.com
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=hotcleaner.com
```

Comment:
I don't see adb detection here.

Reply:
That rule kills this detection (failed to load resource) there are still a lot of placeholders and advertisement name and a cookie in this domain `https://www.hotcleaner.com/cookie-editor/`
`![image](https://user-images.githubusercontent.com/76908410/107516350-425de800-6be7-11eb-90a7-dd81963ce368.png)`

Reply:
Nevermind you fix it. By the way, you missed this, 
<details><summary>Screenshot:</summary>
![image](https://user-images.githubusercontent.com/76908410/107516740-cadc8880-6be7-11eb-923b-350f33bf8bb9.png)
</details><br/>

Reply:
Please use the spoiler for the screenshot.
It was fixed.

Reply:
> By the way, you missed this

Where?
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/107517046-4cb7bc00-6bbe-11eb-8a65-daab598ae52f.png)

</details><br/>

Reply:
I still get it though

Reply:
@zloyden Wait for 10 secs

Reply:
Well, wait for the filters update in 2-3 hours.

Reply:
@zloyden I copied all the rules you added in that commit then navigated to the site.

Reply:
@zloyden I can reproduce this issue with only AdGuard App. With the AdGuard Browser Ad-Blocker it's blocked.

Reply:
Strange, I can reproduce it now. Fixed, thanks for the participation.

Reply:
Why is the app like this though?